Clarify the 1xx and 4xx versioning support (#38685)
dotnet/docs: docs/core/porting/versioning-sdk-msbuild-vs.md

> Targeting `net7.0` is officially supported in Visual Studio 17.4+ only.
> Targeting `net8.0` is officially supported in Visual Studio 17.8+ only.
>

> > <sup>1</<sup>.1xx .NET SDK feature bands are supported throughout the lifecycle of major .NET versions. During the extended support period, support is limited to security fixes and minimal high-priority non-security fixes for Linux only. To learn more about the reasoning for this extended support, refer to this [document](https://github.com/dotnet/source-build#support).

Reviewer: ```suggestion
> <sup>1</<sup>.1xx .NET SDK feature bands are supported throughout the lifecycle of major .NET versions. During the extended support period, support is limited to security fixes and minimal high-priority non-security fixes for Linux only. To learn more about the reasoning for this extended support, see [Source-build support](https://github.com/dotnet/source-build#support).
```

>

> > <sup>2</sup> .4xx .NET SDK feature bands will be supported for the life of the matching runtime as stand-alone installs.

Reviewer: ```suggestion
> <sup>2</sup> .4xx .NET SDK feature bands are supported for the life of the matching runtime as stand-alone installs.
```
